src/components/search-bar-operator-badge-select/SearchBarOperatorBadgeSelect.tsx

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
-import { SSearchBarOperatorBadgeSelect } from './styles/SSearchBarOperatorBadgeSelect';
+import React from "react";
+import { SSearchBarOperatorBadgeSelect } from "./styles/SSearchBarOperatorBadgeSelect";
 
 export interface ISearchBarOperatorBadgeSelectProps {
   hover?: boolean;
   activated?: boolean;
+  onSelect?: () => void;
 }
 
-export const SearchBarOperatorBadgeSelect: React.FC<ISearchBarOperatorBadgeSelectProps> = ({ ...props }) => {
+export const SearchBarOperatorBadgeSelect: React.FC<ISearchBarOperatorBadgeSelectProps> = ({ onSelect, ...props }) => {
   return (
-    <SSearchBarOperatorBadgeSelect {...props}>
+    <SSearchBarOperatorBadgeSelect onClick={onSelect} {...props}>
```

Comment:
[] Make sure to use props destructuring and pass down only required properties. To avoid passing invalid HTML attrs.